common/scala/src/main/scala/whisk/core/WhiskConfig.scala

```diff
@@ -100,7 +100,8 @@ class WhiskConfig(
     val dbPort = this(WhiskConfig.dbPort)
     val dbWhisk = this(WhiskConfig.dbWhisk)
     val dbAuths = this(WhiskConfig.dbAuths)
-    val dbActivations = this(WhiskConfig.dbActivations)
+    lazy val dbSplitActionsAndActivations = this(WhiskConfig.dbSplitActionsAndActivations).toBoolean
```

Comment:
If `dbSplitActionsAndActivations`  is false, the `dbWhisk` is used for activations.
That's the same behaviour like we have today.

```
lazy val dbSplitActionsAndActivations = this(WhiskConfig.dbSplitActionsAndActivations).toBoolean
    lazy val dbActivations = if (dbSplitActionsAndActivations) this(WhiskConfig.dbActivations) else dbWhisk
```